### ISSUE TITLE
Ability to delete items

### ISSUE THREAD

**Issue by jacobvr** [VicShcherbert/HTMLLibrary#9]
We now have an `id` for each `books` item so that we can actually reference that item and work with the array
- [x] In our `renderTable` function you can create `deleteBtn` element, and you can add event listener (google/w3schools) on click on that button
- [x] On click of that button you will run a function as a callback of that event, and this function at first will console.log the `id` of your `element` — see if it works first!
- [x] When you see that you get the id on click of delete button, you can now say `books = books.filter(…)`, thus reassigning `books ` to a filtered array. Read about filtering arrays in JS
- [x] Now instead of `…` in your callback of `.filter` let's create a function that will keep the filtered item if its `id` is NOT equal to `id` of the `element`. that's the core of the function
- [x] Don't forget to call `renderTable` at the end of your event listener callback function because your array now was update
- [x] Also don't forget to update local storage

Basically you are saying, hey `books`, i want you to be assigned to filtered `books`, and it will be filtered based on clicked id: if that id is the same id as of the element, then kick it out; otherwise keep it in the array

**Comment by VicShcherbert** [comment]
It workssssss!! Whoop whoop
